index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;700&display=swap');
+      </style>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main" class="container">
```

Comment:
HTML and CSS best practices:
- Will be less confusing if you avoid adding unused selectors like `container` in this case

Reply:
- [ ] Great working on the inlince comments but I think here it would be great if you explain the reason why you are asking since you are saying it is just confusing but you didn't mention the reason because may be it is obvious to you but not to others 

> You can say 
I Think it would be better if you use `section` tag instaed of `div` tag 🚀 because it is important to make you HTML page semantic and follow the best practice 🚀✅✅🔥🔥